tables/bel.utb

```diff
@@ -34,15 +34,13 @@
 # following definitions have dot 9 set to make them distinguishable
 # from the Latin letters.
 lowercase \x0456 134569  SMALL CYRILLIC LETTER I    і
-always \x0406 134569
+lowercase \x045e 3469  CYRILLIC LETTER SHORT U    ў
+base uppercase \x040e \x045e  CYRILLIC LETTER SHORT U    Ўў
 
 include ru-litbrl.ctb
 
 lowercase \x0456 134569  SMALL CYRILLIC LETTER I    і
 
-lowercase \x045e 3469  CYRILLIC LETTER SHORT U    ў
-base uppercase \x040e \x045e  CYRILLIC LETTER SHORT U    Ўў
-
```

Comment:
What does this change do? Doesn't it need a test?

Reply:
It looks like you already did fixes in bel.utb before, I've just override rules like in uk.utb and kk.utb, no changes in fact.

Reply:
OK I see. I'll add the same comment as in uk.utb to make that clear then.

Reply:
By the way: I know it's well-intended, but I would appreciate it if you would leave the resolving of conversations to the reviewers. It makes it easier for us to keep an overview of which change requests have been addressed and which aren't (according to us).

Reply:
OK, sorry. I thought it's just like a "collapse" button.